subxt/src/backend/mod.rs

```diff
@@ -336,177 +336,308 @@ pub struct StorageResponse {
 #[cfg(test)]
 mod test {
     use super::*;
+    pub use crate::{backend::StorageResponse, error::RpcError};
+    pub use futures::StreamExt;
+    pub use serde::Serialize;
+    pub use std::collections::{HashMap, VecDeque};
+    pub use subxt_core::{config::DefaultExtrinsicParams, Config};
+    pub use tokio::sync::{mpsc, Mutex};
+    pub type RpcResult<T> = Result<T, RpcError>;
+    pub type Item = RpcResult<String>;
+    pub use crate::backend::rpc::RawRpcSubscription;
```

Comment:
I would re-export RawRpcFuture as well here and remove all the prefix `super::rpc::*` below.. see my other comments below

Reply:
done